src/thunderbots/software/test/ai/navigator/util.cpp

```diff
@@ -75,6 +75,40 @@ TEST(NavUtilTest, calculateTransitionSpeedBetweenSegments_tests_nan_corner_cases
         calculateTransitionSpeedBetweenSegments(testp1, testp2, testp3, final_speed)));
 }
 
+TEST(NavUtilTest, convertPointsToMovePrimitives_test)
```

Comment:
I'd like to see a test if no points are passed to the function

Reply:
Good point

Reply:
More like, no point, amirite?

Reply:
oof :cowboy_hat_face: